### ISSUE TITLE
y / yes flags do not function

### ISSUE THREAD

**Issue by MithicSpirit** [achillesrasquinha/pipupgrade#61]
#### What's the bug all about?

Using the `-y` or `--yes` flags result in no packages being upgraded. I am running Ubuntu 18.04 in WSL and pip version 20.0.2 (python 3.6)

#### What command led to this issue?

```
$ pipupgrade --y
```
(same issue if using `--yes` instead)

#### Logs

```
$ pipupgrade -y --verbose
2020-02-22 21:09:35,909 | INFO | Executing command: /home/rpc01/.local/bin/pip --version
2020-02-22 21:09:36,366 | INFO | Executing command: /home/rpc01/.local/bin/pip3 --version
2020-02-22 21:09:36,767 | INFO | Environment: {'version': '1.6.8', 'python_version': '3.6.9', 'os': 'Linux-4.4.0-19041-Microsoft-x86_64-with-Ubuntu-18.04-bionic', 'config': {'path': {'BASE': '/usr/local/lib/python3.6/dist-packages/pipupgrade', 'DATA': '/usr/local/lib/python3.6/dist-packages/pipupgrade/data', 'CACHE': '/home/rpc01/.pipupgrade'}}, 'pip_executables': [{'executable': '/home/rpc01/.local/bin/pip', 'version': 'pip 20.0.2 from /home/rpc01/.local/lib/python3.6/site-packages/pip (python 3.6)'}, {'executable': '/home/rpc01/.local/bin/pip3', 'version': 'pip 20.0.2 from /home/rpc01/.local/lib/python3.6/site-packages/pip (python 3.6)'}], 'settings': {'settings': {'version': '1.6.7', 'cache_timeout': '86400', 'jobs': '12'}}}
2020-02-22 21:09:36,768 | INFO | Arguments Passed: {'verbose': True, 'force': 0, 'ignore_error': 0, 'output': None, 'no_color': 0, 'no_cache': 0, 'no_included_requirements': 0, 'yes': True, 'interactive': 0, 'check': 0, 'user': 0, 'jobs': 12, 'self': False, 'pip': 0, 'all': 0, 'format': 'table', 'latest': 0, 'target_branch': 'master', 'github_username': None, 'github_reponame': None, 'github_access_token': None, 'git_email': None, 'git_username': None, 'pull_request': False, 'project': None, 'pipfile': None, 'requirements': None, 'pip_path': None, 'packages': []}
Checking...
2020-02-22 21:09:36,773 | INFO | `pip` executables found: ['/home/rpc01/.local/bin/pip', '/home/rpc01/.local/bin/pip3']
2020-02-22 21:09:36,773 | INFO | Using 12 jobs...
2020-02-22 21:09:38,149 | INFO | Fetching installed packages for /home/rpc01/.local/bin/pip...
2020-02-22 21:09:38,149 | INFO | Fetching installed packages for /home/rpc01/.local/bin/pip3...
2020-02-22 21:09:38,151 | INFO | Executing command: /home/rpc01/.local/bin/pip list --outdated --format json
2020-02-22 21:09:38,151 | INFO | Executing command: /home/rpc01/.local/bin/pip3 list --outdated --format json
2020-02-22 21:09:46,853 | INFO | 25 packages found for /home/rpc01/.local/bin/pip.
2020-02-22 21:09:46,876 | INFO | 25 packages found for /home/rpc01/.local/bin/pip3.
2020-02-22 21:09:53,008 | INFO | Initializing Package {'name': 'asn1crypto', 'version': '0.24.0', 'latest_version': '1.3.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,008 | INFO | Initializing Package {'name': 'attrs', 'version': '17.4.0', 'latest_version': '19.3.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,008 | INFO | Initializing Package {'name': 'Automat', 'version': '0.8.0', 'latest_version': '20.2.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,010 | INFO | Initializing Package {'name': 'certifi', 'version': '2018.1.18', 'latest_version': '2019.11.28', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,010 | INFO | Initializing Package {'name': 'Automat', 'version': '0.8.0', 'latest_version': '20.2.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,008 | INFO | Initializing Package {'name': 'asn1crypto', 'version': '0.24.0', 'latest_version': '1.3.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,010 | INFO | Initializing Package {'name': 'hyperlink', 'version': '17.3.1', 'latest_version': '19.0.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,010 | INFO | Initializing Package {'name': 'distro-info', 'version': '0.0.0', 'latest_version': '0.10', 'latest_filetype': 'sdist'} of type <class 'dict'>...
2020-02-22 21:09:53,012 | INFO | Fetching package asn1crypto information from DB...
2020-02-22 21:09:53,012 | INFO | Fetching package attrs information from DB...
2020-02-22 21:09:53,011 | INFO | Initializing Package {'name': 'attrs', 'version': '17.4.0', 'latest_version': '19.3.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,012 | INFO | Fetching package Automat information from DB...
2020-02-22 21:09:53,011 | INFO | Initializing Package {'name': 'incremental', 'version': '16.10.1', 'latest_version': '17.5.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,013 | INFO | Fetching package certifi information from DB...
2020-02-22 21:09:53,013 | INFO | Initializing Package {'name': 'certifi', 'version': '2018.1.18', 'latest_version': '2019.11.28', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,015 | INFO | Fetching package Automat information from DB...
2020-02-22 21:09:53,014 | INFO | Initializing Package {'name': 'hyperlink', 'version': '17.3.1', 'latest_version': '19.0.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,014 | INFO | Initializing Package {'name': 'distro-info', 'version': '0.0.0', 'latest_version': '0.10', 'latest_filetype': 'sdist'} of type <class 'dict'>...
2020-02-22 21:09:53,015 | INFO | Initializing Package {'name': 'oauthlib', 'version': '2.0.6', 'latest_version': '3.1.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,015 | INFO | Initializing Package {'name': 'keyring', 'version': '10.6.0', 'latest_version': '21.1.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,015 | INFO | Initializing Package {'name': 'pygobject', 'version': '3.26.1', 'latest_version': '3.34.0', 'latest_filetype': 'sdist'} of type <class 'dict'>...
2020-02-22 21:09:53,016 | INFO | Initializing Package {'name': 'PyJWT', 'version': '1.5.3', 'latest_version': '1.7.1', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,016 | INFO | Initializing Package {'name': 'incremental', 'version': '16.10.1', 'latest_version': '17.5.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,017 | INFO | Fetching package asn1crypto information from DB...
2020-02-22 21:09:53,016 | INFO | Initializing Package {'name': 'jsonschema', 'version': '2.6.0', 'latest_version': '3.2.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,016 | INFO | Initializing Package {'name': 'jsonschema', 'version': '2.6.0', 'latest_version': '3.2.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,017 | INFO | Initializing Package {'name': 'keyring', 'version': '10.6.0', 'latest_version': '21.1.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,018 | INFO | Fetching package hyperlink information from DB...
2020-02-22 21:09:53,018 | INFO | Fetching package distro-info information from DB...
2020-02-22 21:09:53,018 | INFO | Initializing Package {'name': 'pygobject', 'version': '3.26.1', 'latest_version': '3.34.0', 'latest_filetype': 'sdist'} of type <class 'dict'>...
2020-02-22 21:09:53,020 | INFO | Fetching package attrs information from DB...
2020-02-22 21:09:53,018 | INFO | Initializing Package {'name': 'oauthlib', 'version': '2.0.6', 'latest_version': '3.1.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,019 | INFO | Initializing Package {'name': 'PyJWT', 'version': '1.5.3', 'latest_version': '1.7.1', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,021 | INFO | Fetching package incremental information from DB...
2020-02-22 21:09:53,023 | INFO | Fetching package certifi information from DB...
2020-02-22 21:09:53,024 | INFO | Using cached info for package <Package asn1crypto (0.24.0)>.
2020-02-22 21:09:53,026 | INFO | Fetching package hyperlink information from DB...
2020-02-22 21:09:53,025 | INFO | Using cached info for package <Package Automat (0.8.0)>.
2020-02-22 21:09:53,026 | INFO | Using cached info for package <Package attrs (17.4.0)>.
2020-02-22 21:09:53,027 | INFO | Using cached info for package <Package certifi (2018.1.18)>.
2020-02-22 21:09:53,028 | INFO | Fetching package distro-info information from DB...
2020-02-22 21:09:53,028 | INFO | Fetching package oauthlib information from DB...
2020-02-22 21:09:53,029 | INFO | Fetching package keyring information from DB...
2020-02-22 21:09:53,029 | INFO | Fetching package pygobject information from DB...
2020-02-22 21:09:53,030 | INFO | Fetching package PyJWT information from DB...
2020-02-22 21:09:53,030 | INFO | Using cached info for package <Package Automat (0.8.0)>.
2020-02-22 21:09:53,031 | INFO | Fetching package incremental information from DB...
2020-02-22 21:09:53,033 | INFO | Fetching package jsonschema information from DB...
2020-02-22 21:09:53,034 | INFO | Fetching package jsonschema information from DB...
2020-02-22 21:09:53,036 | INFO | Using cached info for package <Package asn1crypto (0.24.0)>.
2020-02-22 21:09:53,037 | INFO | Fetching package keyring information from DB...
2020-02-22 21:09:53,040 | INFO | Fetching package pygobject information from DB...
2020-02-22 21:09:53,043 | INFO | Using cached info for package <Package distro-info (0.0.0)>.
2020-02-22 21:09:53,044 | INFO | Fetching package oauthlib information from DB...
2020-02-22 21:09:53,046 | INFO | Fetching package PyJWT information from DB...
2020-02-22 21:09:53,045 | INFO | Using cached info for package <Package hyperlink (17.3.1)>.
2020-02-22 21:09:53,049 | INFO | Using cached info for package <Package attrs (17.4.0)>.
2020-02-22 21:09:53,051 | INFO | Initializing Package {'name': 'pyOpenSSL', 'version': '17.5.0', 'latest_version': '19.1.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,051 | INFO | Using cached info for package <Package incremental (16.10.1)>.
2020-02-22 21:09:53,053 | INFO | Initializing Package {'name': 'python-debian', 'version': '0.1.32', 'latest_version': '0.1.36', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,053 | INFO | Using cached info for package <Package certifi (2018.1.18)>.
2020-02-22 21:09:53,054 | INFO | Initializing Package {'name': 'pyxdg', 'version': '0.25', 'latest_version': '0.26', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,054 | INFO | Initializing Package {'name': 'PyYAML', 'version': '3.12', 'latest_version': '5.3', 'latest_filetype': 'sdist'} of type <class 'dict'>...
2020-02-22 21:09:53,057 | INFO | Using cached info for package <Package hyperlink (17.3.1)>.
2020-02-22 21:09:53,058 | INFO | Initializing Package {'name': 'pyOpenSSL', 'version': '17.5.0', 'latest_version': '19.1.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,060 | INFO | Using cached info for package <Package distro-info (0.0.0)>.
2020-02-22 21:09:53,060 | INFO | Using cached info for package <Package oauthlib (2.0.6)>.
2020-02-22 21:09:53,060 | INFO | Using cached info for package <Package keyring (10.6.0)>.
2020-02-22 21:09:53,061 | INFO | Using cached info for package <Package pygobject (3.26.1)>.
2020-02-22 21:09:53,061 | INFO | Using cached info for package <Package PyJWT (1.5.3)>.
2020-02-22 21:09:53,062 | INFO | Initializing Package {'name': 'python-debian', 'version': '0.1.32', 'latest_version': '0.1.36', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,062 | INFO | Using cached info for package <Package incremental (16.10.1)>.
2020-02-22 21:09:53,064 | INFO | Using cached info for package <Package jsonschema (2.6.0)>.
2020-02-22 21:09:53,064 | INFO | Initializing Package {'name': 'requests', 'version': '2.18.4', 'latest_version': '2.23.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,065 | INFO | Using cached info for package <Package jsonschema (2.6.0)>.
2020-02-22 21:09:53,066 | INFO | Fetching package pyOpenSSL information from DB...
2020-02-22 21:09:53,066 | INFO | Initializing Package {'name': 'requests-unixsocket', 'version': '0.1.5', 'latest_version': '0.2.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,066 | INFO | Initializing Package {'name': 'pyxdg', 'version': '0.25', 'latest_version': '0.26', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,067 | INFO | Using cached info for package <Package keyring (10.6.0)>.
2020-02-22 21:09:53,067 | INFO | Fetching package python-debian information from DB...
2020-02-22 21:09:53,067 | INFO | Using cached info for package <Package pygobject (3.26.1)>.
2020-02-22 21:09:53,067 | INFO | Initializing Package {'name': 'SecretStorage', 'version': '2.3.1', 'latest_version': '3.1.2', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,068 | INFO | Fetching package pyxdg information from DB...
2020-02-22 21:09:53,068 | INFO | Initializing Package {'name': 'PyYAML', 'version': '3.12', 'latest_version': '5.3', 'latest_filetype': 'sdist'} of type <class 'dict'>...
2020-02-22 21:09:53,068 | INFO | Fetching package PyYAML information from DB...
2020-02-22 21:09:53,069 | INFO | Using cached info for package <Package oauthlib (2.0.6)>.
2020-02-22 21:09:53,070 | INFO | Fetching package pyOpenSSL information from DB...
2020-02-22 21:09:53,070 | INFO | Initializing Package {'name': 'requests', 'version': '2.18.4', 'latest_version': '2.23.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,070 | INFO | Using cached info for package <Package PyJWT (1.5.3)>.
2020-02-22 21:09:53,070 | INFO | Initializing Package {'name': 'requests-unixsocket', 'version': '0.1.5', 'latest_version': '0.2.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,071 | INFO | Initializing Package {'name': 'service-identity', 'version': '16.0.0', 'latest_version': '18.1.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,071 | INFO | Initializing Package {'name': 'ssh-import-id', 'version': '5.7', 'latest_version': '5.8', 'latest_filetype': 'sdist'} of type <class 'dict'>...
2020-02-22 21:09:53,072 | INFO | Fetching package python-debian information from DB...
2020-02-22 21:09:53,072 | INFO | Initializing Package {'name': 'Twisted', 'version': '17.9.0', 'latest_version': '19.10.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,073 | INFO | Initializing Package {'name': 'urllib3', 'version': '1.22', 'latest_version': '1.25.8', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,073 | INFO | Initializing Package {'name': 'SecretStorage', 'version': '2.3.1', 'latest_version': '3.1.2', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,073 | INFO | Fetching package requests information from DB...
2020-02-22 21:09:53,074 | INFO | Initializing Package {'name': 'wheel', 'version': '0.30.0', 'latest_version': '0.34.2', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,075 | INFO | Initializing Package {'name': 'service-identity', 'version': '16.0.0', 'latest_version': '18.1.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,075 | INFO | Fetching package requests-unixsocket information from DB...
2020-02-22 21:09:53,076 | INFO | Fetching package pyxdg information from DB...
2020-02-22 21:09:53,076 | INFO | Using cached info for package <Package pyOpenSSL (17.5.0)>.
2020-02-22 21:09:53,077 | INFO | Fetching package SecretStorage information from DB...
2020-02-22 21:09:53,077 | INFO | Initializing Package {'name': 'ssh-import-id', 'version': '5.7', 'latest_version': '5.8', 'latest_filetype': 'sdist'} of type <class 'dict'>...
2020-02-22 21:09:53,078 | INFO | Initializing Package {'name': 'Twisted', 'version': '17.9.0', 'latest_version': '19.10.0', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,078 | INFO | Fetching package PyYAML information from DB...
2020-02-22 21:09:53,078 | INFO | Using cached info for package <Package python-debian (0.1.32)>.
2020-02-22 21:09:53,079 | INFO | Using cached info for package <Package pyxdg (0.25)>.
2020-02-22 21:09:53,080 | INFO | Initializing Package {'name': 'urllib3', 'version': '1.22', 'latest_version': '1.25.8', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,080 | INFO | Using cached info for package <Package PyYAML (3.12)>.
2020-02-22 21:09:53,080 | INFO | Fetching package requests information from DB...
2020-02-22 21:09:53,081 | INFO | Using cached info for package <Package pyOpenSSL (17.5.0)>.
2020-02-22 21:09:53,081 | INFO | Fetching package requests-unixsocket information from DB...
2020-02-22 21:09:53,082 | INFO | Fetching package service-identity information from DB...
2020-02-22 21:09:53,082 | INFO | Initializing Package {'name': 'wheel', 'version': '0.30.0', 'latest_version': '0.34.2', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,082 | INFO | Fetching package ssh-import-id information from DB...
2020-02-22 21:09:53,083 | INFO | Fetching package Twisted information from DB...
2020-02-22 21:09:53,083 | INFO | Fetching package urllib3 information from DB...
2020-02-22 21:09:53,084 | INFO | Using cached info for package <Package python-debian (0.1.32)>.
2020-02-22 21:09:53,084 | INFO | Fetching package SecretStorage information from DB...
2020-02-22 21:09:53,085 | INFO | Fetching package wheel information from DB...
2020-02-22 21:09:53,085 | INFO | Fetching package service-identity information from DB...
2020-02-22 21:09:53,085 | INFO | Using cached info for package <Package requests (2.18.4)>.
2020-02-22 21:09:53,086 | INFO | Initializing Package {'name': 'zope.interface', 'version': '4.3.2', 'latest_version': '4.7.1', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,087 | INFO | Using cached info for package <Package requests-unixsocket (0.1.5)>.
2020-02-22 21:09:53,087 | INFO | Fetching package ssh-import-id information from DB...
2020-02-22 21:09:53,087 | INFO | Using cached info for package <Package pyxdg (0.25)>.
2020-02-22 21:09:53,088 | INFO | Fetching package Twisted information from DB...
2020-02-22 21:09:53,088 | INFO | Using cached info for package <Package SecretStorage (2.3.1)>.
2020-02-22 21:09:53,090 | INFO | Using cached info for package <Package PyYAML (3.12)>.
2020-02-22 21:09:53,090 | INFO | Fetching package urllib3 information from DB...
2020-02-22 21:09:53,091 | INFO | Initializing Package {'name': 'zope.interface', 'version': '4.3.2', 'latest_version': '4.7.1', 'latest_filetype': 'wheel'} of type <class 'dict'>...
2020-02-22 21:09:53,092 | INFO | Using cached info for package <Package requests (2.18.4)>.
2020-02-22 21:09:53,092 | INFO | Fetching package wheel information from DB...
2020-02-22 21:09:53,093 | INFO | Using cached info for package <Package requests-unixsocket (0.1.5)>.
2020-02-22 21:09:53,094 | INFO | Using cached info for package <Package service-identity (16.0.0)>.
2020-02-22 21:09:53,094 | INFO | Using cached info for package <Package Twisted (17.9.0)>.
2020-02-22 21:09:53,095 | INFO | Using cached info for package <Package ssh-import-id (5.7)>.
2020-02-22 21:09:53,095 | INFO | Using cached info for package <Package urllib3 (1.22)>.
2020-02-22 21:09:53,096 | INFO | Using cached info for package <Package SecretStorage (2.3.1)>.
2020-02-22 21:09:53,096 | INFO | Fetching package zope.interface information from DB...
2020-02-22 21:09:53,096 | INFO | Using cached info for package <Package service-identity (16.0.0)>.
2020-02-22 21:09:53,097 | INFO | Using cached info for package <Package wheel (0.30.0)>.
2020-02-22 21:09:53,098 | INFO | Using cached info for package <Package ssh-import-id (5.7)>.
2020-02-22 21:09:53,099 | INFO | Fetching package zope.interface information from DB...
2020-02-22 21:09:53,099 | INFO | Using cached info for package <Package Twisted (17.9.0)>.
2020-02-22 21:09:53,100 | INFO | Using cached info for package <Package urllib3 (1.22)>.
2020-02-22 21:09:53,101 | INFO | Using cached info for package <Package wheel (0.30.0)>.
2020-02-22 21:09:53,103 | INFO | Using cached info for package <Package zope.interface (4.3.2)>.
2020-02-22 21:09:53,105 | INFO | Using cached info for package <Package zope.interface (4.3.2)>.
2020-02-22 21:09:53,114 | INFO | Packages within `pip` /home/rpc01/.local/bin/pip3 found: [<Package asn1crypto (0.24.0)>, <Package attrs (17.4.0)>, <Package Automat (0.8.0)>, <Package certifi (2018.1.18)>, <Package distro-info (0.0.0)>, <Package hyperlink (17.3.1)>, <Package incremental (16.10.1)>, <Package jsonschema (2.6.0)>, <Package keyring (10.6.0)>, <Package oauthlib (2.0.6)>, <Package pygobject (3.26.1)>, <Package PyJWT (1.5.3)>, <Package pyOpenSSL (17.5.0)>, <Package python-debian (0.1.32)>, <Package pyxdg (0.25)>, <Package PyYAML (3.12)>, <Package requests (2.18.4)>, <Package requests-unixsocket (0.1.5)>, <Package SecretStorage (2.3.1)>, <Package service-identity (16.0.0)>, <Package ssh-import-id (5.7)>, <Package Twisted (17.9.0)>, <Package urllib3 (1.22)>, <Package wheel (0.30.0)>, <Package zope.interface (4.3.2)>]...
2020-02-22 21:09:53,212 | INFO | Packages within `pip` /home/rpc01/.local/bin/pip found: [<Package asn1crypto (0.24.0)>, <Package attrs (17.4.0)>, <Package Automat (0.8.0)>, <Package certifi (2018.1.18)>, <Package distro-info (0.0.0)>, <Package hyperlink (17.3.1)>, <Package incremental (16.10.1)>, <Package jsonschema (2.6.0)>, <Package keyring (10.6.0)>, <Package oauthlib (2.0.6)>, <Package pygobject (3.26.1)>, <Package PyJWT (1.5.3)>, <Package pyOpenSSL (17.5.0)>, <Package python-debian (0.1.32)>, <Package pyxdg (0.25)>, <Package PyYAML (3.12)>, <Package requests (2.18.4)>, <Package requests-unixsocket (0.1.5)>, <Package SecretStorage (2.3.1)>, <Package service-identity (16.0.0)>, <Package ssh-import-id (5.7)>, <Package Twisted (17.9.0)>, <Package urllib3 (1.22)>, <Package wheel (0.30.0)>, <Package zope.interface (4.3.2)>]...
2020-02-22 21:09:53,313 | INFO | Updating registries: [<pipupgrade.model.registry.Registry object at 0x7fee69f787b8>, <pipupgrade.model.registry.Registry object at 0x7fee69f78940>]...
```

**Comment by achillesrasquinha** [comment]
@MithicSpirit the `-y` flag is to avoid answering dialogs and would confirm for each dialog. Are you saying that the packages weren't upgraded because of `y` alone?

**Comment by MithicSpirit** [comment]
Yes. If I run `pipupgrade -y` nothing updates even when running just `pipupgrade` prompts me to update packages, which do get updated if I were to respond `y` to the prompt.

**Comment by achillesrasquinha** [comment]
@MithicSpirit Fixed, released for `1.7.1

**Comment by achillesrasquinha** [comment]
Please reopen if persists. Upgrade as follows `pipupgrade --self`
